force-app/main/default/classes/FlowControlLoopsCollections.cls

```diff
@@ -346,23 +465,35 @@ public with sharing class FlowControlLoopsCollections {
      * @return A list of employee names who have a salary more than 55000.
      */
     public static List<String> getHighPaidEmployees() {
+        
         // Initialize a Map
-        Map<String, Integer> employeeSalaries = new Map<String, Integer>();
+        Map<String, Integer> employees = new Map<String, Integer>();
 
         // Add key-value pairs to the Map
         // add John Doe => 50000
         // add Jane Smith => 60000
         // add Sam Brown => 55000
         // add Alice Johnson => 65000
+        employees.put('John Doe', 50000);
+        employees.put('Jane Smith', 60000);
+        employees.put('Sam Brown', 55000);
+        employees.put('Alice Johnson', 65000);
```

Comment:
Also make sure you are aware of the other way to do this by adding the values right into the constructor.  'John Doe' => 50000,

Reply:
Ya I see that now, thanks will do in future.